src/main/java/net/pms/dlna/DVDISOTitle.java

```diff
@@ -109,80 +158,65 @@ public void run() {
-				}
-				if (line.startsWith("audio stream:")) {
-					DLNAMediaAudio lang = new DLNAMediaAudio();
-					lang.setId(Integer.parseInt(line.substring(line.indexOf("aid: ") + 5, line.lastIndexOf('.')).trim()));
-					lang.setLang(line.substring(line.indexOf("language: ") + 10, line.lastIndexOf(" aid")).trim());
-					int end = line.lastIndexOf(" langu");
-					if (line.lastIndexOf('(') < end && line.lastIndexOf('(') > line.indexOf("format: ")) {
-						end = line.lastIndexOf('(');
-					}
-					lang.setCodecA(line.substring(line.indexOf("format: ") + 8, end).trim());
-					if (line.contains("(stereo)")) {
-						lang.getAudioProperties().setNumberOfChannels(2);
-					} else {
-						lang.getAudioProperties().setNumberOfChannels(6);
+				} else if (line.startsWith("audio stream:")) {
+					DLNAMediaAudio audio = parseMEncoderAudioStream(line);
+					if (audio != null) {
+						audioTracks.add(audio);
 					}
-					audio.add(lang);
-				}
-				if (line.startsWith("subtitle")) {
-					DLNAMediaSubtitle lang = new DLNAMediaSubtitle();
-					lang.setId(Integer.parseInt(line.substring(line.indexOf("): ") + 3, line.lastIndexOf("language")).trim()));
-					lang.setLang(line.substring(line.indexOf("language: ") + 10).trim());
-					if (lang.getLang().equals("unknown")) {
-						lang.setLang(DLNAMediaLang.UND);
+				} else if (line.startsWith("subtitle")) {
+					DLNAMediaSubtitle subtitle = parseMEncoderSubtitleStream(line);
+					if (subtitle != null) {
+						subtitles.add(subtitle);
 					}
-					lang.setType(SubtitleType.UNKNOWN);
-					subs.add(lang);
-				}
-				if (line.startsWith("ID_VIDEO_WIDTH=")) {
+				} else if (line.startsWith("ID_VIDEO_WIDTH=")) {
 					width = line.substring(line.indexOf("ID_VIDEO_WIDTH=") + 15).trim();
-				}
-				if (line.startsWith("ID_VIDEO_HEIGHT=")) {
+				} else if (line.startsWith("ID_VIDEO_HEIGHT=")) {
 					height = line.substring(line.indexOf("ID_VIDEO_HEIGHT=") + 16).trim();
-				}
-				if (line.startsWith("ID_VIDEO_FPS=")) {
+				} else if (line.startsWith("ID_VIDEO_FPS=")) {
 					fps = line.substring(line.indexOf("ID_VIDEO_FPS=") + 13).trim();
-				}
-				if (line.startsWith("ID_LENGTH=")) {
+				} else if (line.startsWith("ID_LENGTH=")) {
 					duration = line.substring(line.indexOf("ID_LENGTH=") + 10).trim();
-				}
-				if (line.startsWith("ID_VIDEO_ASPECT=")) {
+				} else if (line.startsWith("ID_VIDEO_ASPECT=")) {
 					aspect = line.substring(line.indexOf("ID_VIDEO_ASPECT=") + 16).trim();
+				} else if (line.startsWith("ID_VIDEO_FORMAT=")) {
+					String formatStr = line.substring(line.lastIndexOf("=") + 1).trim();
+					if ("0x31435657".equals(formatStr)) {
+						codecV = FormatConfiguration.VC1;
+					} else if ("0x10000001".equals(formatStr)) {
+						codecV = FormatConfiguration.MPEG1;
+					} else if ("0x10000002".equals(formatStr)) {
+						codecV = FormatConfiguration.MPEG2;
+					} else {
+						LOGGER.warn("Unknown video format value \"{}\"", formatStr);
```

Comment:
Just kidding ! Nothing to change !
https://github.com/gjedeer/mplayer-android/blob/master/jni/codecs.conf.h